R/add_difference_row.R

```diff
@@ -0,0 +1,435 @@
+#' Add difference rows
+#'
+#' - [`add_difference_row.tbl_summary()`]
+#'
+#' @param x (`gtsummary`)\cr
+#'   Object with class 'gtsummary'
+#' @param ... Passed to other methods.
+#' @keywords internal
+#'
+#' @author Daniel D. Sjoberg
+#' @export
```

Comment:
if it exported, it should not have the internal keyword, right?

Reply:
Generally, yes. But this is just an S3 generic, and I don't need it on the reference page of the website, so I am hiding it.